### PR TITLE
GoogleAnaryticsのページトラッキングを追加

### PR DESCRIPTION
**Pull request by sawa-zen**
@rrreeeyyy 
お手すきの際にお願いします 🙇

### CODE REVIEW

File: src/js/common/GATracker.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   * @constructor
+   */
+  constructor() {
+    this._visitor = ua('UA-88593825-1');
+  }
+
+  /**
+   * トップページのトラッキングをします。
+   */
+  sendPage() {
```
Comment (rrreeeyyy) [inline src/js/common/GATracker.js]: I think this function name is `sendPageView`, isn't it?

File: src/js/common/GATracker.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    let path = window.location.pathname;
```
Comment (rrreeeyyy) [inline src/js/common/GATracker.js]: I think `document.location.pathname` is more better.

File: src/js/common/GATracker.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    let host = window.location.hostname;
```
Comment (rrreeeyyy) [inline src/js/common/GATracker.js]: I think `document.location.toString()` is more better.

File: src/js/entry.js [hunk 4]
```diff
@@ -1 +1,21 @@
...
+class Entry {
+
+  /**
+   * コンストラクター
+   * @constructor
+   */
+  constructor() {
+    // ページのトラッキング
+    this._tracker = new GATracker();
+    this._tracker.sendPage();
```
Comment (rrreeeyyy) [inline src/js/entry.js]: follow `sendPageView()` if you changed.

File: src/js/top.js [hunk 5]
```diff
@@ -1 +1,21 @@
...
+class Main {
+
+  /**
+   * コンストラクター
+   * @constructor
+   */
+  constructor() {
+    // ページのトラッキング
+    this._tracker = new GATracker();
+    this._tracker.sendPage();
```
Comment (rrreeeyyy) [inline src/js/top.js]: follow `sendPageView()` if you changed.

**Comment by sawa-zen** [comment]
@rrreeeyyy 対応しました :bow:

**Comment by rrreeeyyy** [comment]
LGTM. I'll merge it.

**Pull request merged**